examples/jsm/renderers/webgpu/utils/WebGPUAttributeUtils.js

```diff
@@ -72,9 +72,9 @@ class WebGPUAttributeUtils {
 		const buffer = backend.get( bufferAttribute ).buffer;
 
 		const array = bufferAttribute.array;
-		const updateRange = bufferAttribute.updateRange;
+		const updateRanges = bufferAttribute.updateRanges;
 
```

Comment:
I haven't added 10-release compatibility for WebGPU.

Reply:
That is okay!